packages/chart/src/StatChart.ts

```diff
@@ -0,0 +1,213 @@
+                ];
+                break;
+            case "min_max":
+            default:
+                ticks = [
+                    { label: this.formatQ(0), value: this.quartile(0) },
+                    { label: this.formatQ(1), value: this.quartile(1) },
+                    { label: this.formatQ(2), value: this.quartile(2) },
+                    { label: this.formatQ(3), value: this.quartile(3) },
+                    { label: this.formatQ(4), value: this.quartile(4) }
+                ];
+        }
+
+        const [domainLow, domainHigh] = this.domain(this._selectMode.node().value);
+        return ticks
+            .filter(sd => sd.value >= domainLow && sd.value <= domainHigh)
+            .map(sd => ({ label: sd.label, value: sd.value.toString() }))
+            ;
+    }
+
+    updateBellCurve() {
+        const mode = this._selectMode.node().value;
+        const [domainLow, domainHigh] = this.domain(mode);
+        const padding = (domainHigh - domainLow) * (DOMAIN_PADDING / 100);
+
+        this._bellCurve
+            .xAxisDomainLow(domainLow - padding)
+            .xAxisDomainHigh(domainHigh + padding)
+            .xAxisTicks(this.bellTicks(mode))
+            .data([
+                [this.stdDev(-4), 0],
+                [this.stdDev(-3), 0.3],
+                [this.stdDev(-2), 5],
+                [this.stdDev(-1), 68],
+                [this.stdDev(0), 100],
+                [this.stdDev(1), 68],
+                [this.stdDev(2), 5],
+                [this.stdDev(3), 0.3],
+                [this.stdDev(4), 0]
+            ])
+            .resize({ width: this.width(), height: this.height() - CANDLE_HEIGHT })
+            .render()
+            ;
+    }
+
+    updateCandle() {
+        const candleX = this._bellCurve.dataPos(this.quartile(0));
+        const candleW = this._bellCurve.dataPos(this.quartile(4)) - candleX;
+        this._candle
+            .resize({ width: this.width(), height: CANDLE_HEIGHT })
+            .pos({ x: candleX + candleW / 2, y: CANDLE_HEIGHT / 2 })
+            .width(candleW)
+            .data(this.quartiles())
+            .render()
+            ;
+    }
+
+    update(domNode, element) {
+        super.update(domNode, element);
+        this._tickFormatter = myFormatter(this.tickFormat());
+        this.updateBellCurve();
+        this.updateCandle();
+    }
+}
+export interface StatChart {
+    tickFormat(): string;
+    tickFormat(_: string): this;
+    mean(): number;
+    mean(_: number): this;
+    standardDeviation(): number;
+    standardDeviation(_: number): this;
+    quartiles(): QuertilesType;
+    quartiles(_: QuertilesType): this;
+}
+StatChart.prototype.publish("tickFormat", ".2e", "string", "X-Axis Tick Format");
+StatChart.prototype.publish("mean", 0, "number", "Mean");
+StatChart.prototype.publish("standardDeviation", 0, "number", "Standard Deviation");
+StatChart.prototype.publish("quartiles", [0, 0, 0, 0, 0], "array", "Quartiles");
```

Comment:
For this to work in Dashy it will need to set/get its data via the "data" method as a single row of data with  7 columns (5 quartiles, 1 mean and 1 standardDeviation).

You can keep the `mean`, `standardDeviation`, `quartiles` methods but internally it should set / get from the internal data structure.

Also these methods should not be a PP, just a regular set/get method akin to `data`